### ISSUE TITLE
Generic Object "error 500"

### ISSUE THREAD

**Issue by jmcatellani** [TECLIB/genericobject#59]
Hola, agregue el plugins para modificar los objetos y cuando ingreso me sale el error..

ERROR 500 - INTERNAL SERVER ERROR
Why am I seeing this page?

Is there anything that I can do?

Understanding Filesystem Permissions

How to modify your .htaccess file

How to modify file and directory permissions

**Comment by btry** [comment]
Hi

Unfortunately I don't understand your language. Please explain in english how you triggered this error.

**Comment by jmcatellani** [comment]
hello , install the plugins to manage objects and when I want to enter the options, tools , management of objects I get the following error.

ERROR 500 - INTERNAL SERVER ERROR
Why am I seeing this page?

Is there anything that I can do?

Understanding Filesystem Permissions

How to modify your .htaccess file

How to modify file and directory permissions

**Comment by btry** [comment]
Have a look into the logs of your HTTP server. This may due to file permissions mismatch.

If your server is running as the user www-data, your files needs to be readable by this user. Check the permissions and fix them if needed with chmod

**Comment by btry** [comment]
Hi,

@jmcatellani : can you feedback about your issue please ?

**Comment by jmcatellani** [comment]
Email hello , I hostgator the serivicio GLPI without Chmod can run a Check
permissions , change to full control but still the same error.

Could you help me , I can give you access to my hosting for review If I
have to pay hours no problem.

2015-11-05 11:58 GMT-03:00 btry <notifications@github.com>:

> Hi,
>
> @jmcatellani <https://github.com/jmcatellani> : can you feedback about
> your issue please ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TECLIB/genericobject/issues/59#issuecomment-154084428>
> .
>

-- 

------------------------------

Este mensaje y sus adjuntos se dirigen exclusivamente a su destinatario, 
puede contener información privilegiada o confidencial y es para uso 
exclusivo de la persona o entidad de destino. Si no es usted. el 
destinatario indicado, queda notificado de que la lectura, utilización, 
divulgación y/o copia sin autorización puede estar prohibida en virtud de 
la legislación vigente. Si ha recibido este mensaje por error, le rogamos 
que nos lo comunique inmediatamente por esta misma vía y proceda a su 
destrucción.

**Comment by jmcatellani** [comment]
logs.

[Mon Nov 09 15:31:47 2015] [error] [client 181.30.13.10] File does not
exist:
/home/degasait/public_html/mantenimiento/plugins/genericobject/front/cgi-sys,
referer:
http://www.mantenimiento.degasa.com.ar/plugins/genericobject/front/type.php
[Mon Nov 09 15:31:47 2015] [error] [client 181.30.13.10] File does not
exist:
/home/degasait/public_html/mantenimiento/plugins/genericobject/front/cgi-sys,
referer:
http://www.mantenimiento.degasa.com.ar/plugins/genericobject/front/type.php
[Mon Nov 09 15:31:47 2015] [error] [client 181.30.13.10] File does not
exist:
/home/degasait/public_html/mantenimiento/plugins/genericobject/front/cgi-sys,
referer:
http://www.mantenimiento.degasa.com.ar/plugins/genericobject/front/type.php
[Mon Nov 09 15:31:47 2015] [error] [client 181.30.13.10] File does not
exist:
/home/degasait/public_html/mantenimiento/plugins/genericobject/front/cgi-sys,
referer:
http://www.mantenimiento.degasa.com.ar/plugins/genericobject/front/type.php
[Mon Nov 09 15:31:46 2015] [error] [client 181.30.13.10] File does not
exist:
/home/degasait/public_html/mantenimiento/plugins/genericobject/front/cgi-sys,
referer:
http://www.mantenimiento.degasa.com.ar/plugins/genericobject/front/type.php
[Mon Nov 09 15:31:46 2015] [error] [client 181.30.13.10] File does not
exist:
/home/degasait/public_html/mantenimiento/plugins/genericobject/front/cgi-sys,
referer:
http://www.mantenimiento.degasa.com.ar/plugins/genericobject/front/type.php
[Mon Nov 09 15:31:46 2015] [error] [client 181.30.13.10] SoftException in
Application.cpp:313: File
"/home/degasait/public_html/mantenimiento/plugins/genericobject/front/type.php"
is writeable by group, referer:
http://www.mantenimiento.degasa.com.ar/front/central.php

2015-11-09 18:28 GMT-03:00 Catellani, Juan Martin <jmcatellani@degasa.com.ar
>:

> Email hello , I hostgator the serivicio GLPI without Chmod can run a Check
> permissions , change to full control but still the same error.
>
> Could you help me , I can give you access to my hosting for review If I
> have to pay hours no problem.
>
> 2015-11-05 11:58 GMT-03:00 btry <notifications@github.com>:
>
>> Hi,
>>
>> @jmcatellani <https://github.com/jmcatellani> : can you feedback about
>> your issue please ?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/TECLIB/genericobject/issues/59#issuecomment-154084428>
>> .
>>
>
>

-- 

------------------------------

Este mensaje y sus adjuntos se dirigen exclusivamente a su destinatario, 
puede contener información privilegiada o confidencial y es para uso 
exclusivo de la persona o entidad de destino. Si no es usted. el 
destinatario indicado, queda notificado de que la lectura, utilización, 
divulgación y/o copia sin autorización puede estar prohibida en virtud de 
la legislación vigente. Si ha recibido este mensaje por error, le rogamos 
que nos lo comunique inmediatamente por esta misma vía y proceda a su 
destrucción.

**Comment by jmcatellani** [comment]
check the file and do not have cgi-sys

2015-11-05 11:58 GMT-03:00 btry <notifications@github.com>:

> Hi,
>
> @jmcatellani <https://github.com/jmcatellani> : can you feedback about
> your issue please ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TECLIB/genericobject/issues/59#issuecomment-154084428>
> .
>

-- 

------------------------------

Este mensaje y sus adjuntos se dirigen exclusivamente a su destinatario, 
puede contener información privilegiada o confidencial y es para uso 
exclusivo de la persona o entidad de destino. Si no es usted. el 
destinatario indicado, queda notificado de que la lectura, utilización, 
divulgación y/o copia sin autorización puede estar prohibida en virtud de 
la legislación vigente. Si ha recibido este mensaje por error, le rogamos 
que nos lo comunique inmediatamente por esta misma vía y proceda a su 
destrucción.

**Comment by Sorinnes** [comment]
Hello 

I also have the same error 500 when i try to use Generic Objects...

The error log doesn't tell me anything regarding this. I tried managing the CHMOD cause it looks like an error related to it, but nothing changes.

Any idea please on how to fix this?

Thanks for help!

**Comment by btry** [comment]
Can you show the permissions of files in glpi/plugins/genericobject/inc/ ?

**Comment by Sorinnes** [comment]
All the files inside "inc" folder have 0664 as permissions

**Comment by Sorinnes** [comment]
I changed only the permissions for "inc" unto 0755 as required.

**Comment by btry** [comment]
And it works now ?

**Comment by Sorinnes** [comment]
Nope. It still doesn't work! :-(

**Comment by btry** [comment]
Sure it doent work better : I gave you the wrong path. 

Generic objects generates files for your custom types in glpi/files/_plugins/genericobject/inc 
Have a look here instead of the previous path; sorry for the inconvenience. This path must be readable, and your web server user or group must have read write permissions on the files.

**Comment by Sorinnes** [comment]
I have set public_html/glpi/files/_plugins/genericobject/inc to 0775 permissions.

But still having 500 error! :-(

**Comment by btry** [comment]
Try to su to the user running your web server, and go to the root of your filesystem.

Then cd sub-directory by sub-directory until path where reside your files. In each directory, try a ls to see its content.

**Comment by Sorinnes** [comment]
I am using cpanel to access my web server. And I am able to access each directory.

There's no files though in the directories and sub-directories under public_html/glpi/files/_plugins/genericobject/inc

**Comment by btry** [comment]
if you did not yet create an new itemtype the directory should be empty. then, the error is not a about a class not found, as I supposed.

**Comment by Sorinnes** [comment]
I am able to reproduce (well i think) the same error when i change the CHMOD on the glpi directory itself. When i add group write access to it, i can no longer start glpi, and got a 500 error. So that's why i supposed it has something to deal with this. Otherwise i can't see what would make it not work precisely on my server when it works with others.... :-/

**Comment by Sorinnes** [comment]
I only get a general log on glpi operation, not any specific information about the error. Would the "writeable" rights by group prevent it from working? It wasn't working either when the rights weren't granted for group. But that's the only log i get...

[Thu Dec 03 14:41:28.846055 2015] [:error] [pid 128155] [client 212.166.48.49:49545] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/user.php
[Thu Dec 03 14:41:28.529593 2015] [:error] [pid 131341] [client 212.166.48.49:49541] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/user.php
[Thu Dec 03 14:41:28.212666 2015] [:error] [pid 130135] [client 212.166.48.49:49537] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/user.php
[Thu Dec 03 14:41:26.915170 2015] [:error] [pid 130133] [client 212.166.48.49:49535] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/user.php
[Thu Dec 03 14:41:22.385210 2015] [:error] [pid 131318] [client 212.166.48.49:49421] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/user.php
[Thu Dec 03 14:15:42.516896 2015] [:error] [pid 105653] [client 212.166.48.49:62875] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/dropdown.php
[Thu Dec 03 14:14:40.121267 2015] [:error] [pid 104188] [client 212.166.48.49:62842] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/preference.php
[Thu Dec 03 14:14:38.237083 2015] [:error] [pid 100450] [client 212.166.48.49:62840] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/preference.php
[Thu Dec 03 14:00:59.773888 2015] [:error] [pid 88520] [client 212.166.48.49:62288] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/ticket.form.php?id=1
[Thu Dec 03 13:58:57.324435 2015] [:error] [pid 85031] [client 212.166.48.49:62030] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 13:44:13.333250 2015] [:error] [pid 64466] [client 212.166.48.49:61207] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/ajax" is writeable by group, referer: http://congos.life/glpi/front/central.php
[Thu Dec 03 13:44:12.794437 2015] [:error] [pid 69884] [client 212.166.48.49:61206] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/ajax" is writeable by group, referer: http://congos.life/glpi/front/central.php
[Thu Dec 03 13:44:11.184614 2015] [:error] [pid 70641] [client 212.166.48.49:61201] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/central.php
[Thu Dec 03 13:43:41.607800 2015] [:error] [pid 70341] [client 212.166.48.49:61181] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 13:43:37.373335 2015] [:error] [pid 62080] [client 212.166.48.49:61179] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 13:40:32.737705 2015] [:error] [pid 66109] [client 212.166.48.49:60966] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 12:18:49.089848 2015] [autoindex:error] [pid 1034073] [client 212.166.48.49:56440] AH01276: Cannot serve directory /home/congosli/public_html/glpi/plugins/: No matching DirectoryIndex (index.html.var,index.htm,index.html,index.shtml,index.xhtml,index.wml,index.perl,index.pl,index.plx,index.ppl,index.cgi,index.jsp,index.js,index.jp,index.php4,index.php3,index.php,index.phtml,default.htm,default.html,home.htm,index.php5,Default.html,Default.htm,home.html) found, and server-generated directory index forbidden by Options directive
[Thu Dec 03 12:18:44.762816 2015] [:error] [pid 1034073] [client 212.166.48.49:56438] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/index.php" is writeable by group
[Thu Dec 03 12:11:47.018560 2015] [:error] [pid 1026476] [client 212.166.48.49:55988] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 12:09:46.621003 2015] [:error] [pid 1017908] [client 212.166.48.49:55891] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/profile.form.php?id=3
[Thu Dec 03 12:09:24.418194 2015] [:error] [pid 1013563] [client 212.166.48.49:55873] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/central.php
[Thu Dec 03 12:04:31.118368 2015] [:error] [pid 1017387] [client 212.166.48.49:55731] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/profile.form.php?id=3
[Thu Dec 03 11:47:10.381812 2015] [:error] [pid 994633] [client 212.166.48.49:55363] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/ticket.form.php?id=1
[Thu Dec 03 11:47:08.613877 2015] [:error] [pid 999376] [client 212.166.48.49:55361] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/ticket.form.php?id=1
[Thu Dec 03 11:47:06.057876 2015] [:error] [pid 997007] [client 212.166.48.49:55359] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/ticket.form.php?id=1
[Thu Dec 03 11:47:01.240775 2015] [:error] [pid 997713] [client 212.166.48.49:55357] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/ticket.form.php?id=1
[Thu Dec 03 11:46:24.759768 2015] [:error] [pid 998257] [client 212.166.48.49:55337] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/ticket.form.php?id=1
[Thu Dec 03 11:22:51.800998 2015] [:error] [pid 970558] [client 212.166.48.49:54668] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 11:22:37.343634 2015] [:error] [pid 970248] [client 212.166.48.49:54652] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 11:20:34.271946 2015] [:error] [pid 964929] [client 212.166.48.49:54398] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/fields/field.constant.php" is writeable by group
[Thu Dec 03 11:20:01.471196 2015] [:error] [pid 967029] [client 212.166.48.49:54394] File does not exist: /home/congosli/public_html/glpi/plugins/genericobject/fields/myconstant.php
[Thu Dec 03 11:16:04.392649 2015] [:error] [pid 963266] [client 212.166.48.49:54232] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 11:15:57.439672 2015] [:error] [pid 963266] [client 212.166.48.49:54226] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 11:13:41.411234 2015] [:error] [pid 958847] [client 212.166.48.49:53993] SoftException in Application.cpp:640: Directory "/home/congosli/public_html/glpi/front" is writeable by others, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 11:13:40.531821 2015] [:error] [pid 959938] [client 212.166.48.49:53989] SoftException in Application.cpp:640: Directory "/home/congosli/public_html/glpi/front" is writeable by others, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 11:13:29.647374 2015] [:error] [pid 960262] [client 212.166.48.49:53983] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 11:13:28.609654 2015] [:error] [pid 959938] [client 212.166.48.49:53977] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 11:13:18.255729 2015] [:error] [pid 957950] [client 212.166.48.49:53973] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/plugin.form.php
[Thu Dec 03 11:12:06.539306 2015] [:error] [pid 954151] [client 212.166.48.49:53953] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 11:11:20.902570 2015] [:error] [pid 954788] [client 212.166.48.49:53817] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 10:52:14.684626 2015] [:error] [pid 932317] [client 212.166.48.49:53298] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 10:51:57.444438 2015] [:error] [pid 929501] [client 212.166.48.49:53286] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 10:49:14.421864 2015] [:error] [pid 928300] [client 212.166.48.49:52974] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 10:39:20.643057 2015] [:error] [pid 904892] [client 212.166.48.49:52240] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 10:34:53.137607 2015] [:error] [pid 905942] [client 212.166.48.49:51984] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 10:34:28.365338 2015] [:error] [pid 902506] [client 212.166.48.49:51883] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 10:30:06.416466 2015] [:error] [pid 898362] [client 212.166.48.49:51650] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/central.php
[Thu Dec 03 10:14:50.174786 2015] [:error] [pid 886175] [client 212.166.48.49:51136] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/project.php
[Thu Dec 03 10:14:09.628350 2015] [:error] [pid 886135] [client 212.166.48.49:51004] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/project.php
[Thu Dec 03 10:13:02.527088 2015] [:error] [pid 885490] [client 212.166.48.49:50894] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/central.php
[Thu Dec 03 10:13:01.181091 2015] [:error] [pid 885709] [client 212.166.48.49:50892] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/central.php
[Thu Dec 03 10:12:36.861133 2015] [:error] [pid 885030] [client 212.166.48.49:50886] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/central.php
[Thu Dec 03 10:12:35.160621 2015] [:error] [pid 885209] [client 212.166.48.49:50884] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/central.php
[Thu Dec 03 10:12:33.335249 2015] [:error] [pid 885135] [client 212.166.48.49:50882] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/central.php
[Thu Dec 03 10:12:27.723790 2015] [:error] [pid 885098] [client 212.166.48.49:50878] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/usertitle.form.php
[Thu Dec 03 10:12:25.019489 2015] [:error] [pid 885033] [client 212.166.48.49:50874] SoftException in Application.cpp:631: Directory "/home/congosli/public_html/glpi/front" is writeable by group, referer: http://congos.life/glpi/front/usertitle.form.php
[Thu Dec 03 10:09:51.912644 2015] [:error] [pid 881896] [client 212.166.48.49:50812] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 10:02:17.948250 2015] [:error] [pid 871174] [client 212.166.48.49:50503] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/dropdown.php
[Thu Dec 03 09:56:20.827220 2015] [:error] [pid 866468] [client 212.166.48.49:50155] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/central.php
[Thu Dec 03 09:36:09.046161 2015] [:error] [pid 842277] [client 212.166.48.49:63048] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 09:31:52.221925 2015] [:error] [pid 833077] [client 212.166.48.49:62826] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 09:30:59.175992 2015] [:error] [pid 837285] [client 212.166.48.49:62748] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 09:30:53.274094 2015] [:error] [pid 835200] [client 212.166.48.49:62746] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 09:30:46.489064 2015] [:error] [pid 837246] [client 212.166.48.49:62650] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 09:30:22.762072 2015] [:error] [pid 836028] [client 212.166.48.49:62630] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 09:30:16.155037 2015] [:error] [pid 836593] [client 212.166.48.49:62628] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php
[Thu Dec 03 09:28:38.331977 2015] [:error] [pid 831308] [client 212.166.48.49:62499] SoftException in Application.cpp:261: File "/home/congosli/public_html/glpi/plugins/genericobject/front/type.php" is writeable by group, referer: http://congos.life/glpi/front/plugin.php

**Comment by btry** [comment]
I think suexec is no longer mainstream, but there may be other security systems that may disallow file write to groups. 

If I remember correctly php-fpm mai also disallow acess to group writeable files.

**Comment by Sorinnes** [comment]
Well it doesn't work and i still can't see why.... !!! :'(

**Comment by btry** [comment]
Did you any chmod with the -R option ?

**Comment by Sorinnes** [comment]
Nope. But I just set up ALL the folders with chmod 0755 and it is WORKING NOW!!!!! Can't see the point, but it works and it's all that matters!! ^^

**Comment by btry** [comment]
Well, in my last message I suspected you removed x from folders. without this permission you cannot traverse them.

**Comment by Sorinnes** [comment]
It looks quite having been the opposite. Permissions were set to 0777 and 0775; when certain permissions are granted to group, it doesn't work! And when i remove the said permissions, then it works! It's like there's a check to make sure the folders are not accessible for groups, then it would start. Which is quite weird. But thanks to have tried your very best to help!

**Comment by btry** [comment]
Hi

traversable an writeable is not the same thing. when you chmod'ed 755 your folders you enabled the traversable pemission for anyone. Just don't know if you accidentally disabled it before.

Anyway, the most important is you fixed your issue. Maybe you should check you can create and delete a new type with the plugin.

**Comment by Sorinnes** [comment]
Hi!

I just checked and yes i am able to create new items. Thanks.

No i did not "touch" anything regarding the permission forehand.

I am only left with one issue, i changed some words of the translation in
the .po language file and compiled it unto .mo but nothing changes on the
website !  :-(

I tried cleaning the cache, reloading, etc... Still having the words though
they don't exist anymore in the file!! I thought there was another file
somewhere so i moved the file to another location to find it out, and
apparently it removes the language in the list! Which shows it is indeed
the right file i edited. But despite the word i removed is not anymore in
the .mo file i still have it on the website!!! :-(

But i guess this is not the right place for this issue!

Thanks for coming back to me.
